calibration/experiments/sphere_held_suarez_rhoe_equilmoist/Project.toml

```diff
@@ -0,0 +1,15 @@
+[deps]
+CairoMakie = "13f3f980-e62b-5c42-98c6-ff1f3baf88f0"
+ClimaAnalysis = "29b5916a-a76c-4e73-9657-3c8fd22e65e6"
+ClimaAtmos = "b2c96348-7fb7-4fe0-8da9-78d88439e717"
+ClimaCalibrate = "4347a170-ebd6-470c-89d3-5c705c0cacc2"
+ClimaUtilities = "b3f4f4ca-9299-4f7f-bd9b-81e1242a7513"
+Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
+EnsembleKalmanProcesses = "aa8a2aa5-91d8-4396-bcef-d4f2ec43552d"
+JLD2 = "033835bb-8acc-5ee8-8aae-3f567f8a3819"
+NetCDF = "30363a11-5582-574a-97bb-aa9a979735b9"
+TOML = "fa267f1f-6049-4f14-aa54-33bafae1ed76"
+YAML = "ddb6d928-2868-570f-bddf-ab3f9cf99eb6"
+
+[compat]
+ClimaAtmos = "=0.24.0"
```

Comment:
If ClimaAtmos is pinned to a specific version, why do we need to have a job in buildkite?

Reply:
Yeah, we don't want to pin it if we're checking in the manifest. Alternatively, if we don't check in the manifest, then we could leave it pinned, but that's less reliably reproducible.

Reply:
Since this is experiment a perfect model calibration, the observations will need to be newly generated when the ClimaAtmos version gets updated. I'll keep the pin and remove the manifest.

Reply:
For the buildkite test, I thought we should test the `model_interface.jl` is working as intended since there will no doubt be interface changes for Atmos in the future